src/GitVersionCore/BranchConfigurationCalculator.cs

```diff
@@ -22,7 +20,7 @@ public class BranchConfigurationCalculator
                     currentBranch.FriendlyName));
 
                 var branchConfig = new BranchConfig();
-                ConfigurationProvider.ApplyBranchDefaults(config, branchConfig, "");
+                ConfigurationProvider.ApplyBranchDefaults(config, branchConfig, "", isDevelop: true);
```

Comment:
I removed that with my cleanup. This branch doesn't functionally change anything, it is more about the rename from is-develop to tracks-release-branches